src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,228 @@
 package core.basesyntax;
 
+import java.util.Map;
+import java.util.Objects;
+
+import static core.basesyntax.MyHashMap.Node.hash;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static int CAPACITY = 16;
```

Comment:
CAPACITY should be a constant field. According to the checklist, any magic numbers should be constants. Consider renaming it to `DEFAULT_CAPACITY` and declaring it as `private static final int DEFAULT_CAPACITY = 16;` to follow Java constant naming conventions.